portfolio/src/main/webapp/script.js

```diff
@@ -58,15 +58,15 @@ function setContent() {
 
   /* Use setTimeout to make opacity change from 0 to 1
   after 600ms as a transition */
-  setTimeout(function() {
+  setTimeout(function () {
```

Comment:
Are these spaces recommended/allowed by the style guide? I looked at the whitespace guidelines (https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace) and it doesn't look like it to me.